app/interactors/editions/create_interactor.rb

```diff
@@ -21,38 +19,15 @@ def call
 
   def find_and_lock_edition
     context.edition = Edition.lock.find_current(document: params[:document])
-
-    assert_edition_state(edition, assertion: "can create new edition") do
-      edition.live || edition.discarded?
-    end
-  end
-
-  def create_next_revision
-    updater = Versioning::RevisionUpdater.new(edition.revision, user)
-
-    updater.assign(change_note: "",
-                   update_type: "major",
-                   proposed_publish_time: nil)
-
-    context.next_revision = updater.next_revision
+    context.discarded_edition = Edition.find_by(document: edition.document,
+                                                number: edition.number + 1)
+    assert_edition_state(edition, assertion: "can create new edition") { edition.live }
```

Comment:
We should probably reference somewhere (in this commit message or a separate commit) why we removed the `discarded?` conditional as it's not clear we thought that was done in error.

Reply:
@kevindew Thanks - have updated the commit message on this.